src/core/src/core_logic/PatchInstaller.py

```diff
@@ -343,16 +372,7 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
             self.include_dependencies(package_manager, [package], [version], all_packages, all_package_versions, packages, package_versions, package_and_dependencies, package_and_dependency_versions)
 
             # parent package install (+ dependencies) and parent package result management
-            install_result = Constants.FAILED
-            for i in range(0, Constants.MAX_INSTALLATION_RETRY_COUNT):
-                install_result = package_manager.install_update_and_dependencies_and_get_status(package_and_dependencies, package_and_dependency_versions, simulate)
-
-                if install_result != Constants.INSTALLED:
-                    if i < Constants.MAX_INSTALLATION_RETRY_COUNT - 1:
-                        time.sleep(i + 1)
-                        self.composite_logger.log_warning("Retrying installation of package. [Package={0}]".format(package_manager.get_product_name(package_and_dependencies[0])))
-                else:
-                    break
+            install_result = package_manager.install_update_and_dependencies_and_get_status(package_and_dependencies, package_and_dependency_versions, simulate)
```

Comment:
IIUC, this is a part of the sequential flow, how and why does that change?

Reply:
I forgot to mention regarding this in PR description. Added point (e) in the PR description for this.
I have removed some retries in sequential patching. This is explained below. Please let me know if it is fine to remove the retries.

In sequential patching, for the packages which failed to install in first attempt are retried 3 times. So, at max, there are total 4 attempts in sequential patching for a package. Now, with multi-phase batch patching implementation, we have two attempts in batch patching and 4 attempts in sequential patching. So, at max, there are total 6 attempts for a package. These are too many attempts. Each attempt to install package requires running two commands, one to install package and the other to check if package is installed successfully. These commands take up to total 1 second per package. These commands generate some logs which will add in telemetry throttling and slow down. To reduce attempts, removing 3 retries in sequential patching. So, there are total 2 attempts in batch patching and 1 in sequential patching.